app.js

```diff
@@ -1,21 +1,33 @@
 const express = require("express");
-const { getCategoriesC, getReviewsByIdC, getUsersC, patchReviewVoteC, getReviewsC, getCommentsC, postCommentsC, deleteCommentsC } = require("./controllers/controllers");
+const { getCategoriesC, getReviewsByIdC, getUsersC, patchReviewVoteC, getReviewsC, getCommentsC, postCommentsC, deleteCommentsC, getEndPointC } = require("./controllers/controllers");
 const { status_code_errors, not_found } = require("./controllers/error-handeling");
 
 
 const app = express();
 app.use(express.json());
-
+//TASK 3
 app.get("/api/categories", getCategoriesC);
-app.get("/api/reviews/:review_id", getReviewsByIdC);
-app.get("/api/users", getUsersC);
+//TASK 8 & 11
 app.get("/api/reviews", getReviewsC);
-app.patch("/api/reviews/:review_id", patchReviewVoteC);
+// TASK 5
+app.get("/api/users", getUsersC);
+//TASK 4 & 7
+app.get("/api/reviews/:review_id", getReviewsByIdC);
+// TASK 9
 app.get("/api/reviews/:review_id/comments", getCommentsC);
+// TASK 10
 app.post("/api/reviews/:review_id/comments", postCommentsC);
+// TASK 6
+app.patch("/api/reviews/:review_id", patchReviewVoteC);
+// TASK 12
 app.delete("/api/comments/:comment_id", deleteCommentsC);
+// TASK 13  
+app.get("/api", getEndPointC);
 
+// ERROR CODE OR STATUS HANDELING
```

Comment:
I dont know if any of these comments need to be here, the functions should tell us what they are doing :)